### DISCUSSION TITLE
Connected but no traffic

### DISCUSSION THREAD

**Discussion by Evolve6996** [vpnhood/VpnHood#50] [Q&A]
Hello Trudy thanks for your VPN app 
I managed to run vpnhood on my server and now its running on port 457 because I am using 443 for another VPN program running on my server I also running softether at the same time and both of the other VPNs working just fine.

but vpnhood connected with no traffic goes through, and when I going to check my IP address in whatismyipaddress.com I see my own address instead of the VPN IP address! also client-side diagnosis says everything is good! I didn't activate diagnose mode on the server-side though.
I can give whatever thing you want my VPS pass log and anything thanks!
![vpn](https://user-images.githubusercontent.com/91710620/135557516-574758c6-6280-4f07-86ab-cf90c36a7563.png)

**Comment by trudyhood** [comment]
Hi, @Evolve6996
I see you have selected just 1 country! Maybe there is a bug in our IpFilter. Would you select all countries and let me know the result.

**Reply by Evolve6996** [reply]
thanks for the fast response buddy ❤️ 
now another problem happened.
it now completely shuted down the whole internet when connected 
[log.txt](https://github.com/vpnhood/VpnHood/files/7263905/log.txt)

I am going to sleep now for the next 8 hours was up for the rest of the night 😄 
thanks

**Reply by trudyhood** [reply]
What is your VpnHood version, have you upgraded both client and server?
Can you connect to our Public Server?

**Reply by Evolve6996** [reply]
oh I just downloaded it for windows and used your script to run it on my ubuntu
client Version: 2.1.276.129

ok the public server is connected but I can only ping ips not browsing websites!
 also DNS is working and it will translate domains to ips

**Reply by trudyhood** [reply]
Are you sure you don't have any https or socks proxy on your client browser?
Can you send me the log of diagnose connection to our public server?

**Reply by Evolve6996** [reply]
no, i don't
here is the log

**Reply by Evolve6996** [reply]
ok it's working on android but IDK what is the problem with pc I can connect to other VPns easy in my pc

my windows cannot connect to PPTP SSTP l2pt protocols I don't know if any part of vpnhood related to this btw?

**Comment by trudyhood** [comment]
@Evolve6996 
I inspect the log you sent me, please look at the image, the good news is that VpnHood can tunnel your network, that's why android works, but it looks like something interferes with packets that VpnHood device driver tries to send it **back** to your device (this happen for TCP connections). Perhaps some kind of local firewall installed on your system blocks new incoming connections, or any network inspector apps such as other VPN drivers, VPN, proxies. I think if you a new fresh window should works. Please try with temporary all your local firewalls off (if any). That what’s I could do. Please let me know if you have any ideas and let us know if you find the issue in the future.


![image](https://user-images.githubusercontent.com/72891124/135652281-e2e2d92d-ad6f-4a47-8de4-ba5c3d1526c7.png)

**Reply by Evolve6996** [reply]
thanks for your info ❤️ 

I spent the last day diagnosing this issue on my pc disabled firewalls and antivirus softether-wireguard-amnezia vpn-openvpn and deleted express VPN protonVPN pretty much everything was kinda suspicious revert TCP windows configs in registry back to default but no luck 😢

tested this on UDP and even UDP is not working checked my ethernet adapter setting and found nothing suspicious created a new user on pc as well

**I don't know what is blocking it from working since all of the VPNs from above is ok I removed some windows components with ntlite program and disabled some services you don't have access from the regular services tab in control panel i don't know if its the case caused this or no also, I can't reinstall windows**


I have some side questions I highly appreciated if you answer 🌹 

**is it better to run vpnhood on port 443 for more hiding it from the government firewall since it's the HTTPS port?**


**can I run vpnhood on multiple listener ports at the same time?**



**there is a fake website on generated config **is this like OpenVPN or shadowsocks obfuscation fake website can or is it better to change it to something like one of the local websites not firewalled here for better hiding ?****

**thanks for your time** 👍 ❤️

**Reply by trudyhood** [reply]
I am sorry that you couldn't overcome with it. Your can try to create a new fresh admin user and test VpnHood by it, maybe there is a luck but I am not sure. 
VpnHood UDP protocol, tunnel UDP over UDP and TCP over TCP, but in non UDP, it tunnels all packets over TCP. so VpnHood uses TCP tunnel for TCP connection even in UDP mode.

**is it better to run vpnhood on port 443 for more hiding it from the government firewall since it's the HTTPS port?**
VpnHood default server port is 443. isn't it?!

**there is a fake website on generated config is this like OpenVPN or shadowsocks obfuscation fake website can or is it better to change it to something like one of the local websites not firewalled here for better hiding ?**
Sorry, I really couldn't understand what did you mean, but you can use any certificate or domain on VpnHood server.

**Reply by Evolve6996** [reply]
I am sorry that you couldn't overcome with it. Your can try to create a new fresh admin user and test VpnHood by it, maybe there is a luck but I am not sure. **thanks yes I said I did that**

VpnHood default server port is 443. isn't it?! **Yes it's as I am running it on a different port I just wanted to know if it was your reason when you made the app** 😄

Sorry, I really couldn't understand what did you mean, but you can use any certificate or domain on VpnHood server. **that's ok nevermind** 👍 

sorry, you missed one of my questions **can I run vpnhood server on multiple different ports at the same time?**

**Reply by trudyhood** [reply]
**can I run vpnhood server on multiple different ports at the same time?**
I created an Issue for it and will add this feature in the future release. #52   👍
Meanwhile you can run multi-instance of the VpnHood server.

**Reply by trudyhood** [reply]
Hi @Evolve6996 
**can I run vpnhood server on multiple different ports at the same time?**
Indeed a server may have multiple IPs, and we need to specify a different port on different IPs in case that a port may be used by a different service, so this will be added to VpnHood server. But, I am thinking what's the benefit of listening to **multi ports on a same** IP? For example 90.90.90.90:443, 90.90.90.90:5040!
I think if a firewall blocks HTTPS (443) of an IP, it definitely closes other unknown HTTPS ports too, so listening to **multi ports on the same IP** is useless. let me know if you have any comment.

**Comment by Evolve6996** [comment]
hello again finally fixed it! there were two problems:

1-was checking network connectivity conditions only with pinging and since I blocked all ICMP requests on the server it wasn't responding to anything I didn't have this issue on my other own personal VPNs which are based on Amnezia-VPN and Softether and OpenVPN.
I think vpnhood works differently btw.

2-firewall was also blocking vpnhood but I checked that multiple times only with ping and couldn't find anything wrong, I had to check real website browsing test.

thanks for your help, and again thanks for your amazing job in building this VPN especially its android app 😍 👍

**Reply by trudyhood** [reply]
We have released a new version 2.2.283, now you can config the server so it can listen to more than one ports. by the way, I am not sure it can solve your issue
